app-backend/migrations/src/main/scala/migrations/Summary.scala

```diff
@@ -157,5 +157,6 @@ object MigrationSummary {
   M156
   M157
   M158
+  M159
```

Comment:
It looks like we are at migration `160` now. So I think the one in this PR should be `161`?

Reply:
james skipped 159 to do 160

Reply:
![](https://media2.giphy.com/media/lgyURtXOtFGZa/giphy.gif?cid=3640f6095c4f69da693439756b4664a1)